### DISCUSSION TITLE
Outline buttons not working

### DISCUSSION THREAD

**Discussion by naro-code-lab** [akveo/react-native-ui-kitten#1524] [Q&A]
Hi i noticed changes in my outline buttons, initially the code below worked for me but not any more

```javascript
  <Button
  appearance="outline"
  size="medium"
  style={{ backgroundColor: "transparent" }}
  >
          <Text
            allowFontScaling={false}
            style={[
              {
                fontWeight: "500",
                fontFamily: "Poppins-Medium",
              },
              {
                color:
                  colorScheme == "dark"
                    ? "#FFFFFF"
                    : theme["color-primary-500"],
              },
            ]}
          >
            Custom
          </Text>
  </Button>
```

**Comment by fessor10** [comment]
> Hi i noticed changes in my outline buttons, initially the code below worked for me but not any more
> 
> 
> 
> ```javascript
> 
>   <Button
> 
>   appearance="outline"
> 
>   size="medium"
> 
>   style={{ backgroundColor: "transparent" }}
> 
>   >
> 
>           <Text
> 
>             allowFontScaling={false}
> 
>             style={[
> 
>               {
> 
>                 fontWeight: "500",
> 
>                 fontFamily: "Poppins-Medium",
> 
>               },
> 
>               {
> 
>                 color:
> 
>                   colorScheme == "dark"
> 
>                     ? "#FFFFFF"
> 
>                     : theme["color-primary-500"],
> 
>               },
> 
>             ]}
> 
>           >
> 
>             Custom
> 
>           </Text>
> 
>   </Button>
> 
> ```
> 
> 

Any help with this?
